src/app/routes/assets/asset-list/asset-list.component.ts

```diff
@@ -275,6 +275,7 @@ export class AssetListComponent implements OnInit, OnDestroy {
   private mapEnrolments() {
     return (source: Observable<Asset[]>) => {
       return source.pipe(
+        filter(assets => assets.length > 0),
```

Comment:
Is there a chance of a null exception here?
```suggestion
        filter(assets => assets?.length > 0),
```

Reply:
I don't think so, because we always get here a list. Empty or with assets, but i can add elvis operator as you suggested.